core/src/test/java/org/elasticsearch/search/aggregations/bucket/histogram/InternalDateHistogramTests.java

```diff
@@ -55,7 +55,7 @@ protected InternalDateHistogram createTestInstance(String name,
                                                        List<PipelineAggregator> pipelineAggregators,
                                                        Map<String, Object> metaData,
                                                        InternalAggregations aggregations) {
-        int nbBuckets = randomInt(10);
+        int nbBuckets = randomNumberOfBuckets();
```

Comment:
do you mean changing to 10 just here or also for all multi bucket aggs (see `InternalMultiBucketAggregationTestCase.DEFAULT_MAX_NUMBER_OF_BUCKETS`) ?